src/ingest/src/main/scala/osmesa/ingest/Ingest.scala

```diff
@@ -122,21 +126,45 @@ object IngestApp extends CommandApp(
 
       val df = ss.read.orc(orc)
 
-      val ppnodes = ProcessOSM.preprocessNodes(df)
-      val ppways = ProcessOSM.preprocessWays(df)
-      val nodeGeoms = ProcessOSM.constructPointGeometries(ppnodes)
-      val wayGeoms = ProcessOSM.reconstructWayGeometries(ppnodes, ppways)
-      println("PRIOR TO WAY/NODE UNION")
-      nodeGeoms.withColumn("minorVersion", lit(null).cast(IntegerType)).printSchema
-      wayGeoms.printSchema
+      println("caching...")
+      val cache = Option(new URI(cacheDir).getScheme) match {
+        case Some("file") => Caching.onFs(cacheDir)
+        case Some("s3") => Caching.onS3(cacheDir)
+        case _ => Caching.none
+      }
```

Comment:
See https://github.com/azavea/osmesa/pull/47/files#diff-9399197c9828d222bc4d93480ee442f8 for a fix for local FS caching.